File: code_examples/3-Structured_Problem_Solving/11-Compound_Boolean/C/main.c
// Copyright (c) 2020 Mr. Coxall All rights reserved.
//
// Created by: Mr. Coxall
// Created on: Sep 2020
// This program uses a compound boolean statement

#include <stdio.h>

int main() {
    // this function uses a compound boolean statement
    int termMark;
    int projectMark;

    // input
    printf("Enter term mark (as %%): ");
    scanf("%d", &termMark);
    printf("Enter project mark (as %%): ");
    scanf("%d", &projectMark);
    printf("\n");

    // process & output
    if (termMark >= 50 && projectMark >= 50) {
        printf("You passed the course.\n");
    } else {
        printf("You did not pass the course.\n");
    }

    printf("\nDone.\n");
    return 0;
}

File: code_examples/3-Structured_Problem_Solving/11-Compound_Boolean/JavaScript/main.js
/* Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program uses a compound boolean statement
 */

const prompt = require("prompt-sync")()

// input
const termMark = parseInt(prompt("Enter term mark (as %): "));
const projectMark = parseInt(prompt("Enter project mark (as %): "));
console.log("");

// process & output
if (termMark >= 50 && projectMark >= 50) {
  console.log("You passed the course.");
} else {
  console.log("You did not pass the course.");
}

console.log("\nDone.");
